saphir/src/server.rs

```diff
@@ -615,3 +618,18 @@ mod ssl_loading_utils {
         Ok(ders)
     }
 }
+
+/// Inject a http request into saphir
+pub async fn inject_request(req: RawRequest<RawBody>) -> Result<RawResponse<RawBody>, SaphirError> {
```

Comment:
I would recommend renaming this function `inject_raw` instead of `inject_request`. This is for consistency since request refer explicitly to `Saphir::Request` and we will probable add an api to inject non-raw request (for benchmark / test purpose)

Reply:
I called it like this first than I changed my mind. Finally, it was a good idea.  inject_raw or inject_raw_request ?

Reply:
`inject_raw`

Reply:
done